yroots/subdivision.py

```diff
@@ -20,15 +20,15 @@
 from itertools import product
 from matplotlib import pyplot as plt
 from scipy.linalg import lu
-import itertools
 import time
 import warnings
+from numba import jit
 
 def solve(funcs, a, b, rel_approx_tol=1.e-15, abs_approx_tol=1.e-12,
           max_cond_num=1e5, good_zeros_factor=100, min_good_zeros_tol=1e-5,
           check_eval_error=True, check_eval_freq=1, plot=False,
           plot_intervals=False, deg=None, target_deg=None, max_level=999,
-          return_potentials=False, method='svd', target_tol=1.e-15):
+          return_potentials=False, method='svd', target_tol=1.e-16):
```

Comment:
Why did we make target_tol tighter? If we can be that accurate without slowing down that's great

Reply:
I noticed that while I was experimenting with target_tol that setting it tighter has given us greater accuracy without very much slowdown. I think this is because get_abs_approx_tol overrides the default target_tol when it becomes necessary to do so, so a tighter default at worst changes little and at best gives us greater accuracy. I can (and probably should, so I will) look into more of what the optimal target_tol should be.